cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableTest.java

```diff
@@ -89,18 +91,24 @@
   @BeforeClass
   public static void beforeClass() throws Exception {
     cConf = CConfiguration.create();
-    hBaseTableUtil = new HBaseTableUtilFactory(cConf).get();
+    hBaseTableUtil = new HBaseTableUtilFactory(cConf, new SimpleNamespaceQueryAdmin()).get();
     // TODO: CDAP-1634 - Explore a way to not have every HBase test class do this.
-    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(), NAMESPACE1);
-    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(), NAMESPACE2);
+    hBaseTableUtil.createNamespaceIfNotExists(TEST_HBASE.getHBaseAdmin(),
```

Comment:
Id.Namespace doesn't really contain any more info than just the namespace itself. As I mentioned before, CDAP-6534 will change how we convert CDAP Namespace to HBase entities. That will most likely change the way we do the mapping.